src/UI/Pages/CompileReports/Components/StatusLabel.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { CompileStatus } from "@/Core";
+import { Spacer } from "@/UI/Components";
+import { StatusLabel } from "./StatusLabel";
+
+export default {
+  title: "CompileReport/StatusLabel",
+  component: StatusLabel,
+};
+
+export const Default = () => (
```

Comment:
The progress spinner should be visible in storybook as well.
But that will automatically be fixed.